x/epochstorage/README.md

```diff
@@ -0,0 +1,172 @@
+# `x/epochstorage`
+
+## Abstract
+
+This document specifies the epochstorage module of Lava Protocol.
+
+Lava blocks are divided into bulk of blocks called epochs. epoch length is defines as a parameter of the module.
+The module keeps track of the epochs in memory, epochs that needs to be deleted and anchering data to each epoch, mainly providers stakes.
```

Comment:
needs -> need
and what is "anchering"? do you mean anchoring?
and note -> Note (sentence start)
will take effect -> will be applied

Reply:
done